.github/workflows/deploy.yml

```diff
@@ -20,10 +20,14 @@ jobs:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
 
-    - name: Clone apisix's docs from repo
+    - name: Clone apisix's docs from repo & update image's link
       run: |
         git clone https://github.com/apache/apisix.git
+        cp -r apisix/doc/images ./website/static && rm -rf apisix/doc/images
         cp -r apisix/doc ./docs/apisix
+        cd ./docs/apisix
+        find . -type f -name "*.md" -print0 | xargs -0 sed -i '' -e 's/\.\.\/images/https\:\/\/apisix\.apache\.org\/images/g'
```

Comment:
I'm sure there will have easier ways to handle this case, but I have no idea how to do that.